CMakeLists.txt

```diff
@@ -3,34 +3,60 @@ cmake_minimum_required(VERSION 3.6)
 
 project(Ashura)
 
+
```

Comment:
can you remove the cmake changes since it's no longer needed to build on mac, right?

Reply:
@lamarrr The fix is from only the part where I conditionally added SPIRV for non-apple build